.gitignore

```diff
@@ -1,4 +1,3 @@
-package-lock.json
```

Comment:
hold up, this might be a good thing to have, no?

Reply:
done